process/tac-member-R&Rs.md

```diff
@@ -2,23 +2,52 @@
-- Regular dialogue with Technical Initative leads
-- Provide technical review of funding requests for Technical Initatives
+- Participate in Technical Initiatives regularly to ensure alignment with the Technical Vision and MVSR of OpenSSF, and to identify any resource or funding requirements
+- Regular dialogue with Technical Initiative leads
+- Provide technical review of funding requests for Technical Initiatives
 - Coordinate technical community matters related to the success of Technical Initiatives and the mission of the OpenSSF, including in support of end-users and ambassadors;
 - Participate in community events to raise awareness of OpenSSF tools and standards among project maintainers
 - Plans to engage & recruit community members 
 - Be available and responsive to OpenSSF staff for consulting early and often on operational plans,  issues and communications that have direct or indirect impact on OpenSSF community
 - Be available and responsive to leaders of “adjacent” initiatives (i.e. Alpha/Omega, GNU Toolchain) for early and often changes in strategy, roadmap, and press releases, etc
 
 - TAC and Technical Initiative meetings shall be open, public meetings. For special circumstances, the TAC may hold meetings limited to the TAC voting representatives, invited guests, and LF staff
-- Annually elect a Chairperson who will preside over meetings, set the agenda for meetings, ensure meeting minutes are taken, and serve on the Governing Board as the TAC’s representative (the “TAC Representative”)
-  
 - Time commitment: average 5+ hours per week
 - Must inform the TAC if unable to continue fulfilling committed Technical Initiative sponsor role, so that a new sponsor can be identified and transitioned into the role.
+
+## TAC Chair
+
+This section is primarily derived from [OpenSSF charter][charter] 5c, vi:
+
+> The role of the TAC is to structure and facilitate collaboration among the Technical Initiatives. The TAC will be responsible for:
+>
+> vi. annually electing a chairperson to preside over meetings, set the agenda for meetings, ensure meeting minutes are taken, and who will also serve on the Governing Board as the TAC's representative (the "TAC Representative")
+
+### Additional Chair Roles
+
+- TAC Voting Representative for the OpenSSF Governing Board (GB)
+- TAC Voting Representative for the OpenSSF [Governance Committee][gov-comm]
+
+### Responsibilities
+
+- Fulfill TAC member responsibilities
+- Fulfill the responsibilities of the TAC Chair's [additional roles](#additional-chair-roles)
+- Preside over meetings
+- Set the agenda for meetings
+- Ensure meeting minutes are taken
+- Coordinate the TAC to conduct technical management and oversight of OpenSSF Technical Initiatives
+- Organize the selection of a TAC member to serve on the OpenSSF [Governance Committee][gov-comm], *in addition to the TAC Chair's membership*
```

Comment:
```suggestion
```

Removing this, following a discussion in the private channel.
i.e., 

> The selection process of the TAC Members will be determined by the TAC (via a vote if necessary). However one TAC Member out of the 2 must be the TAC Chair (the TAC Chair seat does not count towards the Governing Boards seven seats).

ref: https://github.com/ossf/Governance-Committee/blob/main/CHARTER.md#membership